src/ray/gcs/gcs_client/global_state_accessor.cc

```diff
@@ -374,33 +374,87 @@ std::string GlobalStateAccessor::GetSystemConfig() {
   return future.get();
 }
 
-ray::Status GlobalStateAccessor::GetNodeToConnectForDriver(
-    const std::string &node_ip_address, std::string *node_to_connect) {
+ray::Status GlobalStateAccessor::GetAliveNodes(std::vector<rpc::GcsNodeInfo> &nodes) {
```

Comment:
```suggestion
ray::Status GlobalStateAccessor::GetAliveNodes(std::vector<rpc::GcsNodeInfo> *nodes) {
```

We use pointer for output

Reply:
Discussed offline, this follows C++ style guide.